### ISSUE TITLE
ReduceOnly do not work

### ISSUE THREAD

**Issue by Timmen** [JKorf/Bybit.Net#142]
Hello. I have issue with sending reduce-only orders. It do not work for one-side mode nor for hedge mode.

I receive '10001: Request parameter error.' in any case, even if reduceOnly: false. For test I create BUY position with 1 quantity.

```
_bybitClient.V5Api.Trading.PlaceOrderAsync(Category.Linear,
                                           symbol: symbol,
                                           side: OrderSide.Sell,
                                           type: NewOrderType.Market,
                                           quantity: 1,
                                           positionIdx: Bybit.Net.Enums.V5.PositionIdx.OneWayMode,
                                           reduceOnly: true)
```

With hedge mode the same thing (actually I need to do this in hedge mode)

```
_bybitClient.V5Api.Trading.PlaceOrderAsync(Category.Linear,
                                           symbol: symbol,
                                           side: OrderSide.Sell,
                                           type: NewOrderType.Market,
                                           quantity: 1,
                                           positionIdx: Bybit.Net.Enums.V5.PositionIdx.BuyHedgeMode,
                                           reduceOnly: true)
```

Am I doing something wrong?

**Comment by wclark17** [comment]
I just figured out what is going on with this.  It is a very annoying thing, but bybit apparently doesn't like the captital T of F in True or False.   
I put this patch in my local copy, which is obviously a bad hack and I am sure @JKorf can code it better, but it got it working for me;

In BybitRestClientApiTrading.cs, line 79 & 80 I changed to:

```
            parameters.AddOptionalParameter("reduceOnly", reduceOnly.HasValue ? (reduceOnly.Value ?"true" : "false") : null);
            parameters.AddOptionalParameter("closeOnTrigger", closeOnTrigger.HasValue ? (closeOnTrigger.Value ? "tr
```

This allowed my orders to go in.
